src/Surv.jl

```diff
@@ -363,6 +363,42 @@ function _tabulate_surv(T, Δ)
             noutcomes = noutcomes)
 end
 
+# Expand the time points in 'stats' to include all times in 'times'.
+function _padstats(stats, times)
```

Comment:
I don't understand what this does and I'm not sure it works? See comment below in tests

Reply:
Please also consider a more informative name so it's clear for developers what to use this for